### ISSUE TITLE
Please provide obj.data and obj.names

### ISSUE THREAD

**Issue by sriks6711** [achen353/Taiwanese-Traffic-Object-Detection#4]
To make the command [1] run successfully, we also need obj.data and obj.names.

[1] $ ./darknet detector demo data/obj.data cfg/yolov4-tiny-obj.cfg yolov4-tiny-obj_best.weights <video>...

**Comment by sriks6711** [comment]
> To make the command [1] run successfully, we also need obj.data and obj.names.
> 
> [1] $ ./darknet detector demo data/obj.data cfg/yolov4-tiny-obj.cfg yolov4-tiny-obj_best.weights

@jaydubai While only @achen353 can provide them, here is a placeholder working somewhat -

1) obj.data (a simple text file with the following contents)
classes=4
names=data/obj.names

2) obj.names (a simple text file with the following contents which contains the 4 classes)
vehicle
scooter
pedestrian
bicycle

**Comment by jaydubal** [comment]
Thanks, still not working on my end. Even when using the obj.names you suggested. 
I guess the main problem is the missing .cfg
Which .cfg are you using?

**Comment by mayank-xinthe** [comment]
Below are the changes that we have made for successful execution.

Few Modifications we have Made:

    A. In File cfg/yolov4-tiny-custom.cfg:
        1. Replace filters=255 to filters=27 at line 212 and 361
        2. Replace classes=80 to classes=4 at line 218 and 267

    B. In File cfg/coco.data:
        1. Replace classes= 80 to classes= 4 at line 1

    C. In File data/coco.names:
        Replace the file contents as below:
            vehicle
            scooter
            pedestrian
            bicycle

After above modifications are made,  ececute the below command:
    ./darknet detector demo cfg/coco.data cfg/yolov4-tiny-custom.cfg backup/yolov4-tiny-obj_best.weights sample_input video eg: ABC.mp4

**Comment by jaydubal** [comment]
Yes it is working now... the line 361 is actually 263 in .cfg file
